background.js

```diff
@@ -1,128 +1,132 @@
-				break;
-			default:
-				console.log(request);
-		}
-	});
-	
-if(initStorage("v0.8.92", true)) {
-	// v0.7
-	initStorage("popupcolor", "blue");
-	initStorage("highlight", true);
-	
-	// v0.8
-	// No changes to options
-	
-	// V0.8.5
-	initStorage("textboxhl", false);
-
-	// v0.8.6
-	initStorage("onlyreading", false);
-	// v0.8.8
-	if (localStorage['highlight'] == "yes")
-		localStorage['highlight'] = "true";
-	if (localStorage['highlight'] == "no")
-		localStorage['highlight'] = "false";
-	if (localStorage['textboxhl'] == "yes")
-		localStorage['textboxhl'] = "true";
-	if (localStorage['textboxhl'] == "no")
-		localStorage['textboxhl'] = "false";
-	if (localStorage['onlyreading'] == "yes")
-		localStorage['onlyreading'] = "true";
-	if (localStorage['onlyreading'] == "no")
-		localStorage['onlyreading'] = "false";
-	initStorage("copySeparator", "tab");
-	initStorage("maxClipCopyEntries", "7");
-	initStorage("lineEnding", "n");
-	initStorage("minihelp", "true");
-	initStorage("disablekeys", "false");
-	initStorage("kanjicomponents", "true");
-
-	for (i = 0; i*2 < rcxDict.prototype.numList.length; i++) {
-		initStorage(rcxDict.prototype.numList[i*2], "true")
-	}
-
-    // v0.8.92
-	initStorage("popupDelay", "150");
-	initStorage("showOnKey", "");
-}
-
-/** 
-* Initializes the localStorage for the given key. 
-* If the given key is already initialized, nothing happens. 
-* 
-* @author Teo (GD API Guru)
-* @param key The key for which to initialize 
-* @param initialValue Initial value of localStorage on the given key 
-* @return true if a value is assigned or false if nothing happens 
-*/ 
-function initStorage(key, initialValue) {
-  var currentValue = localStorage[key];
-  if (!currentValue) {
-	localStorage[key] = initialValue;
-	return true;
-  }
-  return false;
-}
-
-rcxMain.config = {};
-rcxMain.config.css = localStorage["popupcolor"];
-rcxMain.config.highlight = localStorage["highlight"];
-rcxMain.config.textboxhl = localStorage["textboxhl"];
-rcxMain.config.onlyreading = localStorage["onlyreading"];
-rcxMain.config.copySeparator = localStorage["copySeparator"];
-rcxMain.config.maxClipCopyEntries = localStorage["maxClipCopyEntries"];
-rcxMain.config.lineEnding = localStorage["lineEnding"];
-rcxMain.config.minihelp = localStorage["minihelp"];
-rcxMain.config.popupDelay = parseInt(localStorage["popupDelay"]);
-rcxMain.config.disablekeys = localStorage["disablekeys"];
-rcxMain.config.showOnKey = localStorage["showOnKey"];
-rcxMain.config.kanjicomponents = localStorage["kanjicomponents"];
-rcxMain.config.kanjiinfo = new Array(rcxDict.prototype.numList.length/2);
-for (i = 0; i*2 < rcxDict.prototype.numList.length; i++) {
-	rcxMain.config.kanjiinfo[i] = localStorage[rcxDict.prototype.numList[i*2]];
-}
-
+chrome.browserAction.onClicked.addListener(rcxMain.inlineToggle);
+chrome.tabs.onSelectionChanged.addListener(rcxMain.onTabSelect);
+chrome.runtime.onMessage.addListener(
+    function (request, sender, response) {
```

Comment:
oh no!

You changed all the tabs into spaces which makes every single line a diff...

The main goal was to get your intermediate changes on github so I'll merge this into a different branch so that we can iterate on it.

Reply:
Ah, actually this PR is already a branch; let me know if you can fix the spacing easily or if you mind if I push changes to it.

Reply:
Taking a look now.